### PR TITLE
ILMT, Release notes and related changes

### PR DESCRIPTION
**Pull request by veena-ux**
Release notes and related changes

### CODE REVIEW

**Comment by LightGuard** [comment]
At the current time, there is no point in working in the 9.x branch. Work should be done in main and back-ported to the 9.x branch.

File: installation/ilmt-track-bamlcns-kubernetes.adoc [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+
+== Prerequisites
+Install the License Service by following the https://www.ibm.com/docs/en/cloud-paks/foundational-services/3.23?topic=software-installing-license-service-stand-alone[tutorial].
+
+== Tracking BAMOE License
+After installing BAMOE Canvas and its dependencies following the installation tutorial, run the following commands depending on where BAMOE Canvas was installed.
+
+* OpenShift
+----
```
Comment (LightGuard) [inline installation/ilmt-track-bamlcns-kubernetes.adoc:12]: [source, shell]

That should be added above line 12 to provide the proper highlighting.

File: installation/ilmt-track-bamlcns-kubernetes.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+oc patch deployment <bamoe-canvas_deployment_name> -p \
+  '{"spec": {"template":{"metadata":{"annotations":{"productID":"46b14c7f2f894218a7879b3f6416024f", "productName": "IBM Decision Manager Open Edition", "productMetric": "VIRTUAL_PROCESSOR_CORE", "productChargedContainers": "All"}}}} }'
+----
+  
+* Kubernetes
+----
```
Comment (LightGuard) [inline installation/ilmt-track-bamlcns-kubernetes.adoc:18]: Same for highlighting here.

File: installation/ilmt-track-bamlcns-kubernetes.adoc [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+kubectl patch deployment <bamoe-canvas_deployment_name> -p \
+  '{"spec": {"template":{"metadata":{"annotations":{"productID":"46b14c7f2f894218a7879b3f6416024f", "productName": "IBM Decision Manager Open Edition", "productMetric": "VIRTUAL_PROCESSOR_CORE", "productChargedContainers": "All"}}}} }'
+----
+  Where <bamoe-canvas_deployment_name> is the name of the Deployment resource created when you installed BAMOE Canvas.
```
Comment (LightGuard) [inline installation/ilmt-track-bamlcns-kubernetes.adoc:22]: Probably use `<bamoe-canvas_deployment_name>` (in backticks) to set it off better.

File: installation/ilmt-track-bamlcns-kubernetes.adoc [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+* Alternative
+
+If you have installed BAMOE Canvas by using .yaml resources, the deployment resource must be edited to add the following lines:
+----
```
Comment (LightGuard) [inline installation/ilmt-track-bamlcns-kubernetes.adoc:27]: [source, yaml]

File: installation/ilmt-track-bamlcns-metrictool.adoc [hunk 5]
```diff
@@ -0,0 +1,42 @@
+= Tracking BAMOE License use in IBM License Metric Tool
+include::../partials/attributes.adoc[]
+//tag::ilmt-overview-description[]
+You can maintain an up-to-date inventory of software assets that are installed in your infrastructure, gather information about your hardware, and ensure license compliance of your enterprise by using ILMT. 
+You get to know the softwares that are existing in your system, and where or how it is used. 
+By monitoring consumption of PVU, VPC, and RVU MAPC metrics by IBM® products under full and subcapacity licensing terms, you can understand the current license consumption. 
+You can also predict the future license consumption, thus providing the means to optimize your license costs.
+//end::vscode-extensions-description[]
```
Comment (LightGuard) [inline installation/ilmt-track-bamlcns-metrictool.adoc:8]: Why the tag here? Also, there should be a new line after the include.

File: installation/ilmt-track-bamlcns-metrictool.adoc [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+== Configuring IBM License Metric Tool
+
+IBM® License Metric Tool generates reports to help you maintain compliance with your Virtual Processor Core (VPC) subcapacity license terms. 
+License Metric Tool can calculate VPC consumption only when software identification tags exist and are activated by the customer. 
+The optman global option licenseType must be set to perServer (default value) to activate Virtual Processor Core consumption tracking.
```
Comment (LightGuard) [inline installation/ilmt-track-bamlcns-metrictool.adoc:14]: Typically we make the codes and source options stand out a bit more by using backticks:
The `optman` global option `licenseType` must be set to `perServer`...

File: installation/ilmt-track-bamlcns-metrictool.adoc [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+By monitoring consumption of PVU, VPC, and RVU MAPC metrics by IBM® products under full and subcapacity licensing terms, you can understand the current license consumption. 
+You can also predict the future license consumption, thus providing the means to optimize your license costs.
+//end::vscode-extensions-description[]
+
+== Configuring IBM License Metric Tool
+
+IBM® License Metric Tool generates reports to help you maintain compliance with your Virtual Processor Core (VPC) subcapacity license terms. 
+License Metric Tool can calculate VPC consumption only when software identification tags exist and are activated by the customer. 
+The optman global option licenseType must be set to perServer (default value) to activate Virtual Processor Core consumption tracking. 
+You can also set the optman global option licenseType to byWorkstation to specify that the licensing type (either perServer or perJob) is specified at creation time for each workstation.
```
Comment (LightGuard) [inline installation/ilmt-track-bamlcns-metrictool.adoc:15]: Same idea here.

File: installation/ilmt-track-bamlcns-metrictool.adoc [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+
+License Metric Tool automatically detects the following IBM Business Automation Manager Open Editions 9.0.x release chargeable components that are part of the product:
```
Comment (LightGuard) [inline installation/ilmt-track-bamlcns-metrictool.adoc:18]: We have variables for the full name and the version: `{PRODUCT_LONG}` and `{VERSION}`

File: installation/ilmt-track-bamlcns-metrictool.adoc [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+
+|=== 
+| Chargeable Software Components
+
+|	Process Automation Manager Open Edition
+|	Decision Manager Open Edition
```
Comment (LightGuard) [inline installation/ilmt-track-bamlcns-metrictool.adoc:24]: The extra whitespace in the table cells should not be there.

File: installation/ilmt-track-bamlcns-metrictool.adoc [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+
+The following IBM Business Automation Manager chargeable components require manual deployment of software tags, for example 9.0.1:
+
+|=== 
+| Chargeable Software Components | Assigned Software tags
+
+|	Process Automation Manager Open Edition
+|   ibm.com_IBM_Process_Automation_Manager_Open_Edition-9.0.1.swidtag
+|	Decision Manager Open Edition
+|   ibm.com_IBM_Decision_Manager_Open_Edition-9.0.1.swidtag
```
Comment (LightGuard) [inline installation/ilmt-track-bamlcns-metrictool.adoc:38]: Same here. Backticks should also surround the tag names.

File: installation/ilmt-track-bamlcns-metrictool.adoc [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+License Metric Tool automatically detects the following IBM Business Automation Manager Open Editions 9.0.x release chargeable components that are part of the product:
+
+|=== 
+| Chargeable Software Components
+
+|	Process Automation Manager Open Edition
+|	Decision Manager Open Edition
+|===
+
+To detect and count remote nodes that are managed by IBM Business Automation Manager Open Editions chargeable components, you must manually deploy software tags because no product code is present on those nodes.
```
Comment (LightGuard) [inline installation/ilmt-track-bamlcns-metrictool.adoc:27]: You can use the `{PRODUCT_LONG}` variable here too.

File: installation/ilmt-track-bamlcns-metrictool.adoc [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+|=== 
+| Chargeable Software Components | Assigned Software tags
+
+|	Process Automation Manager Open Edition
+|   ibm.com_IBM_Process_Automation_Manager_Open_Edition-9.0.1.swidtag
+|	Decision Manager Open Edition
+|   ibm.com_IBM_Decision_Manager_Open_Edition-9.0.1.swidtag
+|===
+
+NOTE: The IBM Business Automation Manager Open Editions 9.0 license can be used for all the IBM Business Automation Manager access methods and application plug-ins available on the Automation Hub and its 9.0.x fix releases.
```
Comment (LightGuard) [inline installation/ilmt-track-bamlcns-metrictool.adoc:41]: Variables here too. If we need to create a new variable for `9.0.x` you can add one in partials/attributes.adoc

File: installation/release-notes-main/release-notes-9-0-0.adoc
Comment (LightGuard) [inline installation/release-notes-main/release-notes-9-0-0.adoc:1]: Release notes and ILMT should be in different pull requests to keep the changes atomic.

Comment (LightGuard) [inline installation/release-notes-main/release-notes-9-0-0.adoc:1]: If you're moving the release notes section, the old files should be removed.

File: installation/release-notes-main/release-notes-9-0-1.adoc
Comment (LightGuard) [inline installation/release-notes-main/release-notes-9-0-1.adoc:1]: Same comments here.

File: installation/release-notes-main/release-notes-9-0-1.adoc [hunk 13]
```diff
@@ -0,0 +1,57 @@
...
+- <<New in this release>>
+- <<System requirements>>
+- <<Fix lists>>
+- <<Known limitations>>
+- <<Download documents>>
+
+
+== Description
+
+IBM Business Automation Manager Open Editions, Version  fix release, provides important bug fixes requested by customers and the security fixes to PSIRTs/CVEs.
```
Comment (LightGuard) [inline installation/release-notes-main/release-notes-9-0-1.adoc:19]: {PRODUCT_LONG} and version variables should go here.

File: installation/release-notes-main/release-notes-9-0-1.adoc [hunk 14]
```diff
@@ -0,0 +1,57 @@
...
+    
+- Documented the ILMT compliance and reporting of customer usage for BAMOE Version 9.0.1
+- Documented instructions for patch update from 9.0.0 to 9.0.1
+- Delivered Migration guide for Version 9.0
+- Delivered Security bulletin for BAMOE canvas and runtime libraries
+- Support for Quarkus 2.16.12
+
+== System requirements
+
+This section highlights the supported version updates in Version 9.0.1 fix release. You can access the detailed system requirements from the IBM Support site link for {PRODUCT} 9.0.1 fix release.
```
Comment (LightGuard) [inline installation/release-notes-main/release-notes-9-0-1.adoc:31]: Sentence per line.

File: installation/release-notes-main/release-notes-9-0-1.adoc [hunk 15]
```diff
@@ -0,0 +1,57 @@
...
+See https://spcr-prod-preview.dal1a.cirrus.ibm.com/software/reports/compatibility/clarity-reports/report/html/softwareReqsForProduct?deliverableId=839878FFE5974B07B85E83D0B1D368B0&osPlatforms=Linux%7CWindows&duComponentIds=S012%7CS011%7CD014%7CC013&mandatoryCapIds=25&optionalCapIds=71%7C26%7C1
+
+
+== Fix lists
+The summary list of fixes that are included in Version 9.0.1 fix release.
+
+BAMOE canvas
```
Comment (LightGuard) [inline installation/release-notes-main/release-notes-9-0-1.adoc:38]: Maybe a third level title here?

File: installation/release-notes-main/release-notes-9-0-1.adoc [hunk 16]
```diff
@@ -0,0 +1,57 @@
...
+
+- git-cors-proxy-image and kie-sandbox-image on macOS ARM processors
+- MvelCompiler is not coercing type double apache or incubator-kie-issues
+- Usage of appropriate BigDecimal method to convert to primitive type apache or incubator-kie-issues
+- BAMOE Canvas image uses the root filesystem on the container to store its assets and configuration files
+- BAMOE 9.0.0 runtime image fails to run on Kubernetes with readOnlyRootFilesystem: true due to swidtag creation
+
+Licensing Model update
```
Comment (LightGuard) [inline installation/release-notes-main/release-notes-9-0-1.adoc:46]: Same here.

File: installation/release-notes-main/release-notes-9-0-1.adoc [hunk 17]
```diff
@@ -0,0 +1,57 @@
...
+
+- ILMT Libraries does not work on Native builds
```
Comment (LightGuard) [inline installation/release-notes-main/release-notes-9-0-1.adoc:48]: Native probably doesn't need to be capitalized.

File: installation/release-notes-main/release-notes-9-0-1.adoc [hunk 18]
```diff
@@ -0,0 +1,57 @@
...
+
+== Known limitations
+
+To review the most current known problems in detail, see Known limitations in IBM Business Automation Management Open Editions 9.0.1 fix release. As the limitations are discovered and resolved, the IBM Support team updates the knowledge base.
```
Comment (LightGuard) [inline installation/release-notes-main/release-notes-9-0-1.adoc:52]: Do we have a link for this? Also the variables here would be good to use.

Comment (LightGuard) [inline installation/release-notes-main/release-notes-9-0-1.adoc:52]: Sentence per line.

File: installation/release-notes-main/release-notes-9-0-1.adoc [hunk 19]
```diff
@@ -0,0 +1,57 @@
...
+
+== Download documents
+
+The document contains information for obtaining IBM Business Automation Manager Open Editions (BAMOE) 9.0.0, earlier release and its fix releases, currently 9.0.1. BAMOE versions are designated with a numerical Major.Minor.Patch following Open Source community Semantic Versioning principle. Fixes for on-premises releases are available through IBM Fix Central, and BAMOE canvas container images are available through Quay Container Registry.
```
Comment (LightGuard) [inline installation/release-notes-main/release-notes-9-0-1.adoc:56]: Variables and sentence per line.

Comment (LightGuard) [inline installation/release-notes-main/release-notes-9-0-1.adoc:56]: on-premise installs versus on-premises releases?

Comment (LightGuard) [inline installation/release-notes-main/release-notes-9-0-1.adoc:56]: Variable for BAMOE Canvas

**Review by LightGuard — CHANGES_REQUESTED** [review]

File: installation/ilmt-track-bamlcns-kubernetes.adoc [hunk 20]
```diff
@@ -0,0 +1,37 @@
+= Tracking BAMOE license use in Kubernetes
+include::../partials/attributes.adoc[]
+
+
+== Prerequisites
+Install the License Service by following the https://www.ibm.com/docs/en/cloud-paks/foundational-services/3.23?topic=software-installing-license-service-stand-alone[tutorial].
```
Comment (porcelli) [inline installation/ilmt-track-bamlcns-kubernetes.adoc:6]: Is the only public reference about License Service is CloudPak docs?

A lot of effort has been done to communicate customers that BAMOE is not part of CloudPak, and now this might confuse users.

File: installation/ilmt-track-bamlcns-kubernetes.adoc [hunk 21]
```diff
@@ -0,0 +1,37 @@
...
+* Kubernetes
+----
+kubectl patch deployment <bamoe-canvas_deployment_name> -p \
+  '{"spec": {"template":{"metadata":{"annotations":{"productID":"46b14c7f2f894218a7879b3f6416024f", "productName": "IBM Decision Manager Open Edition", "productMetric": "VIRTUAL_PROCESSOR_CORE", "productChargedContainers": "All"}}}} }'
+----
+  Where <bamoe-canvas_deployment_name> is the name of the Deployment resource created when you installed BAMOE Canvas.
+
+* Alternative
+
+If you have installed BAMOE Canvas by using .yaml resources, the deployment resource must be edited to add the following lines:
```
Comment (porcelli) [inline installation/ilmt-track-bamlcns-kubernetes.adoc:26]: variables missing

File: installation/ilmt-track-bamlcns-metrictool.adoc (hunk 11 above)
Comment (porcelli) [inline installation/ilmt-track-bamlcns-metrictool.adoc:27]: this content is confusing to me... what remote nodes? running what?

File: installation/ilmt-track-bamlcns-metrictool.adoc (hunk 10 above)
Comment (porcelli) [inline installation/ilmt-track-bamlcns-metrictool.adoc:38]: nothing manual is needed. there're required libraries that automatically generates the swidtags.  This is related to the ILMT libraries content that I sent to you @veena-ux to be incorporated.

File: installation/release-notes-main/release-notes-9-0-0.adoc [hunk 22]
```diff
@@ -0,0 +1,77 @@
...
+*** Kubernetes
+** Run on containerized environments with new product image 
+* IBM Business Automation Manager Open Editions Developer Tools
+** Available on the marketplace
+* Runtime support for DMN 1.4
+* Accelerators
+* Upstream-first 
+** Upstream certified versions available through new IBM Bills Of Material (dependency management)
+* Cloud
+** Helm Charts
```
Comment (porcelli) [inline installation/release-notes-main/release-notes-9-0-0.adoc:48]: Helm is not included

Comment (porcelli) [inline installation/release-notes-main/release-notes-9-0-0.adoc:48]: where this list came from?

File: installation/release-notes-main/release-notes-9-0-0.adoc [hunk 23]
```diff
@@ -0,0 +1,77 @@
...
+
+== Deprecated
+
+* Red Hat Business Automation VSCode Bundle
```
Comment (porcelli) [inline installation/release-notes-main/release-notes-9-0-0.adoc:52]: makes no sense

File: installation/release-notes-main/release-notes-9-0-0.adoc [hunk 24]
```diff
@@ -0,0 +1,77 @@
...
+
+== Removed Components
+* OpenShift Business Automation Operator
+* Business Central
+* KIE Server
+* Smart Router
+* Process Instance Migration Tool
+* Dashbuilder Runtime
+* Process Migration Service
+* OpenShift Images
```
Comment (porcelli) [inline installation/release-notes-main/release-notes-9-0-0.adoc:62]: this makes little sense to me, this list is about v8.. while v9 is a completely different architecture and runtime stack.

File: installation/release-notes-main/release-notes-9-0-0.adoc [hunk 25]
```diff
@@ -0,0 +1,77 @@
...
+* Embeddable editors
+* {GITHUB_CHROME_EXTENSION_LINK}[*{GITHUB_CHROME_EXTENSION}*]
+
+== Versions
+* Kogito 1.40.0.Final
+* Jakarta EE 9
+* Quarkus 2.16.7.Final
+
+== Decision Service Extensions
+* org.kie.kogito
```
Comment (porcelli) [inline installation/release-notes-main/release-notes-9-0-0.adoc:74]: makes no sense to me

File: installation/release-notes-main/release-notes-9-0-1.adoc (hunk 15 above)
Comment (porcelli) [inline installation/release-notes-main/release-notes-9-0-1.adoc:38]: variable

File: installation/release-notes-main/release-notes-9-0-1.adoc [hunk 26]
```diff
@@ -0,0 +1,57 @@
...
+
+
+== Fix lists
+The summary list of fixes that are included in Version 9.0.1 fix release.
+
+BAMOE canvas
+
+- git-cors-proxy-image and kie-sandbox-image on macOS ARM processors
+- MvelCompiler is not coercing type double apache or incubator-kie-issues
+- Usage of appropriate BigDecimal method to convert to primitive type apache or incubator-kie-issues
```
Comment (porcelli) [inline installation/release-notes-main/release-notes-9-0-1.adoc:42]: these are not related to canvas

File: installation/release-notes-main/release-notes-9-0-1.adoc [hunk 27]
```diff
@@ -0,0 +1,57 @@
...
+- BAMOE Canvas image uses the root filesystem on the container to store its assets and configuration files
+- BAMOE 9.0.0 runtime image fails to run on Kubernetes with readOnlyRootFilesystem: true due to swidtag creation
```
Comment (porcelli) [inline installation/release-notes-main/release-notes-9-0-1.adoc:44]: this is not related to canvas

**Review by porcelli — CHANGES_REQUESTED** [review]
+1 for all @LightGuard input + a few additional ones that requires a few updates.

thank you @veena-ux for helping give us a better docs shape!

File: installation/release-notes-main/release-notes-9-0-0.adoc
Comment (veena-ux) [reply installation/release-notes-main/release-notes-9-0-0.adoc:1]: This will be created next time.

Comment (veena-ux) [reply installation/release-notes-main/release-notes-9-0-0.adoc:1]: I have retained the existing release notes in the new structure. The structure looks like this...
Release notes
    Release notes 9.0.1
    Release notes 9.0.0

File: installation/release-notes-main/release-notes-9-0-1.adoc
Comment (veena-ux) [reply installation/release-notes-main/release-notes-9-0-1.adoc:1]: Retained the previous release notes and not removing it.

File: installation/release-notes-main/release-notes-9-0-1.adoc (hunk 18 above)
Comment (veena-ux) [reply installation/release-notes-main/release-notes-9-0-1.adoc:52]: No link available as of now.

**Review by veena-ux — COMMENTED** [review]
Taken care of the comments.

File: installation/release-notes-main/release-notes-9-0-1.adoc (hunk 26 above)
Comment (veena-ux) [reply installation/release-notes-main/release-notes-9-0-1.adoc:43]: May be I will remove the BAMOE canvas title from there

File: installation/ilmt-track-bamlcns-kubernetes.adoc (hunk 20 above)
Comment (porcelli) [reply installation/ilmt-track-bamlcns-kubernetes.adoc:6]: it's what it's :)

File: installation/release-notes-main/release-notes-9-0-0.adoc (hunk 24 above)
Comment (porcelli) [reply installation/release-notes-main/release-notes-9-0-0.adoc:62]: k, I didn't see the diffs in that way... but if the content was already there.. ok.

@LightGuard can you confirm? :)

File: installation/ilmt-track-bamlcns-metrictool.adoc [hunk 28]
```diff
@@ -0,0 +1,42 @@
...
+You get to know the softwares that are existing in your system, and where or how it is used. 
+By monitoring consumption of PVU, VPC, and RVU MAPC metrics by IBM® products under full and subcapacity licensing terms, you can understand the current license consumption. 
+You can also predict the future license consumption, thus providing the means to optimize your license costs.
+
+
+== Configuring IBM License Metric Tool
+
+IBM® License Metric Tool generates reports to help you maintain compliance with your Virtual Processor Core (VPC) subcapacity license terms. 
+License Metric Tool can calculate VPC consumption only when software identification tags exist and are activated by the customer. 
+The 'optman' global option 'licenseType' must be set to 'perServer' (default value) to activate Virtual Processor Core consumption tracking.
```
[the remark this answers is not in the document]
Comment (porcelli) [reply installation/ilmt-track-bamlcns-metrictool.adoc:14]: I have no idea about those terms =/

File: installation/ilmt-track-bamlcns-metrictool.adoc [hunk 29]
```diff
@@ -0,0 +1,42 @@
...
+License Metric Tool automatically detects the following {PRODUCT_LONG} {VERSION90X} release chargeable components that are part of the product:
+
+|=== 
+|Chargeable Software Components
+
+|Process Automation Manager Open Edition
+|Decision Manager Open Edition
+|===
+
+To detect and count remote nodes that are managed by {PRODUCT_LONG} chargeable components, you must manually deploy software tags because no product code is present on those nodes.
```
[the remark this answers is not in the document]
Comment (porcelli) [reply installation/ilmt-track-bamlcns-metrictool.adoc:27]: please, as I mentioned this makes no sense

File: installation/ilmt-track-bamlcns-metrictool.adoc (hunk 28 above)
[the remark this answers is not in the document]
Comment (LightGuard) [reply installation/ilmt-track-bamlcns-metrictool.adoc:14]: Also unknown to me. Was this info initially pulled from somewhere else?

File: installation/ilmt-track-bamlcns-metrictool.adoc [hunk 30]
```diff
@@ -0,0 +1,42 @@
...
+
+== Configuring IBM License Metric Tool
+
+IBM® License Metric Tool generates reports to help you maintain compliance with your Virtual Processor Core (VPC) subcapacity license terms. 
+License Metric Tool can calculate VPC consumption only when software identification tags exist and are activated by the customer. 
+The 'optman' global option 'licenseType' must be set to 'perServer' (default value) to activate Virtual Processor Core consumption tracking. 
+You can also set the 'optman' global option 'licenseType' to byWorkstation to specify that the licensing type (either 'perServer' or 'perJob') is specified at creation time for each workstation.
+
+
+License Metric Tool automatically detects the following {PRODUCT_LONG} {VERSION90X} release chargeable components that are part of the product:
```
[the remark this answers is not in the document]
Comment (LightGuard) [reply installation/ilmt-track-bamlcns-metrictool.adoc:18]: Honestly, I would be surprised if they're different in this scenario.

File: installation/ilmt-track-bamlcns-metrictool.adoc [hunk 31]
```diff
@@ -0,0 +1,42 @@
...
+|Process Automation Manager Open Edition
+|Decision Manager Open Edition
+|===
+
+To detect and count remote nodes that are managed by {PRODUCT_LONG} chargeable components, you must manually deploy software tags because no product code is present on those nodes. 
+License Metric Tool generates and assigns dedicated software tags during the creation of the Readiness Package for the latest release of {PRODUCT_LONG}.
+
+The following {PRODUCT_LONG} chargeable components require manual deployment of software tags, for example {VERSION901}:
+
+|===
```
[the remark this answers is not in the document]
Comment (LightGuard) [reply installation/ilmt-track-bamlcns-metrictool.adoc:32]: I believe so.

File: installation/release-notes-main/release-notes-9-0-0.adoc (hunk 24 above)
Comment (LightGuard) [reply installation/release-notes-main/release-notes-9-0-0.adoc:62]: Please see my comment above for this whole file.

File: installation/release-notes-main/release-notes-9-0-0.adoc
Comment (LightGuard) [reply installation/release-notes-main/release-notes-9-0-0.adoc:1]: This file used to exist at introduction/release-notes.adoc. I do not know why it is being moved in this Pull Request. It should not.

File: installation/release-notes-main/release-notes-9-0-0.adoc (hunk 22 above)
Comment (LightGuard) [reply installation/release-notes-main/release-notes-9-0-0.adoc:48]: This slipped through the cracks in 9.0.0. We said it was included in 9.0.0 :)

File: installation/release-notes-main/release-notes-9-0-0.adoc (hunk 23 above)
Comment (LightGuard) [reply installation/release-notes-main/release-notes-9-0-0.adoc:52]: For the 9.0.0 release notes?

File: installation/release-notes-main/release-notes-9-0-0.adoc (hunk 24 above)
Comment (LightGuard) [reply installation/release-notes-main/release-notes-9-0-0.adoc:62]: This PR is moving the release notes to a different folder structure. I propose they be moved to a different PR.

File: partials/attributes.adoc [hunk 32]
```diff
@@ -7,6 +7,8 @@
 :PRODUCT_LONG: IBM Business Automation Manager Open Editions
 :OPENSHIFT: OpenShift
 :VERSION: 9.0.0
+:VERSION901: 9.0.1
+:VERSION90X : 9.0.x
```
Comment (LightGuard) [inline partials/attributes.adoc:11]: For consistency, please use VERSION_901 and VERSION_90X.

File: partials/attributes.adoc [hunk 33]
```diff
@@ -7,6 +7,8 @@
...
 :PRODUCT_VERSION: {VERSION}
 :VERSION_OLD: 8.0
```
Comment (LightGuard) [inline partials/attributes.adoc:13]: This may need to change to VERSION_8.0

File: installation/release-notes-main/release-notes-9-0-1.adoc [hunk 34]
```diff
@@ -0,0 +1,74 @@
...
+- <<System requirements>>
+- <<Fix lists>>
+- <<Known limitations>>
+- <<Download documents>>
+- <<Upgrade {PRODUCT_SHORT} from Version 9.0.0 to Version 9.0.1>>
+
+
+== Description
+
+{PRODUCT_LONG}, {VERSION901}  fix release, provides important bug fixes requested by customers and the security fixes to PSIRTs/CVEs.
```
Comment (porcelli) [inline installation/release-notes-main/release-notes-9-0-1.adoc:20]: I'd remove "PSIRTs/" leaving just CVEs

File: installation/release-notes-main/release-notes-9-0-1.adoc [hunk 35]
```diff
@@ -0,0 +1,74 @@
...
+== Upgrade {PRODUCT_SHORT} from Version 9.0.0 to Version 9.0.1
+
+To upgrade {PRODUCT_SHORT} from Version 9.0.0 to Version 9.0.1, follow these steps:
+
+- Update QUARKUS Version to {QUARKUS_VERSION_SHORT_12_FINAL}
+- Update pom file from their projects with Version 9.0.1 {PRODUCT_SHORT} BOM
+- For direct library components, use Version 8.40.1 for drools or Version 1.40.2 for kogito libraries.
+- Rebuild projects and re-deploy as regular software rollout.
+
+For Canvas:
```
Comment (porcelli) [inline installation/release-notes-main/release-notes-9-0-1.adoc:71]: use variable for canvas

File: installation/release-notes-main/release-notes-9-0-1.adoc [hunk 36]
```diff
@@ -0,0 +1,74 @@
...
+
+The new containers are the drop-in replacements for the existing. 
+You can use https://docs.openshift.com/container-platform/4.14/applications/deployments/route-based-deployment-strategies.html#deployments-blue-green_route-based-deployment-strategies[blue/green deployment]
```
Comment (porcelli) [inline installation/release-notes-main/release-notes-9-0-1.adoc:74]: "You can use blue/green deployment" missing the word in the end: "strategies."

**Review by porcelli — CHANGES_REQUESTED** [review]
almost there!

File: installation/release-notes-main/release-notes-9-0-1.adoc (hunk 19 above)
Comment (veena-ux) [reply installation/release-notes-main/release-notes-9-0-1.adoc]: Fixed

File: installation/release-notes-main/release-notes-9-0-1.adoc (hunk 18 above)
Comment (veena-ux) [reply installation/release-notes-main/release-notes-9-0-1.adoc]: Fixed

File: installation/ilmt-track-bamlcns-kubernetes.adoc (hunk 21 above)
Comment (veena-ux) [reply installation/ilmt-track-bamlcns-kubernetes.adoc]: Fixed

File: installation/ilmt-track-bamlcns-metrictool.adoc (hunk 11 above)
Comment (veena-ux) [reply installation/ilmt-track-bamlcns-metrictool.adoc]: Removed and updated

File: installation/ilmt-track-bamlcns-metrictool.adoc (hunk 10 above)
Comment (veena-ux) [reply installation/ilmt-track-bamlcns-metrictool.adoc]: Content updated

File: installation/release-notes-main/release-notes-9-0-0.adoc (hunk 22 above)
Comment (veena-ux) [reply installation/release-notes-main/release-notes-9-0-0.adoc]: Removed

Comment (veena-ux) [reply installation/release-notes-main/release-notes-9-0-0.adoc]: This is in 9.0.0 release (previous release). Left it as is except Helm.

File: installation/release-notes-main/release-notes-9-0-0.adoc (hunk 23 above)
Comment (veena-ux) [reply installation/release-notes-main/release-notes-9-0-0.adoc]: Content removed and updated

File: installation/release-notes-main/release-notes-9-0-0.adoc (hunk 24 above)
Comment (veena-ux) [reply installation/release-notes-main/release-notes-9-0-0.adoc]: Yes, existing content and is in previous release 9.0.0

File: installation/release-notes-main/release-notes-9-0-0.adoc (hunk 25 above)
Comment (veena-ux) [reply installation/release-notes-main/release-notes-9-0-0.adoc]: Content updated

File: installation/release-notes-main/release-notes-9-0-1.adoc (hunk 15 above)
Comment (veena-ux) [reply installation/release-notes-main/release-notes-9-0-1.adoc]: Fixed

File: installation/release-notes-main/release-notes-9-0-1.adoc (hunk 26 above)
Comment (veena-ux) [reply installation/release-notes-main/release-notes-9-0-1.adoc:42]: Removed and updated

File: installation/release-notes-main/release-notes-9-0-1.adoc (hunk 27 above)
Comment (veena-ux) [reply installation/release-notes-main/release-notes-9-0-1.adoc]: Yes, updated

File: installation/ilmt-track-bamlcns-metrictool.adoc (hunk 28 above)
[the remark this answers is not in the document]
Comment (veena-ux) [reply installation/ilmt-track-bamlcns-metrictool.adoc]: Deleted and updated

File: installation/ilmt-track-bamlcns-metrictool.adoc (hunk 30 above)
[the remark this answers is not in the document]
Comment (veena-ux) [reply installation/ilmt-track-bamlcns-metrictool.adoc:11]: Content updated as per Alex

File: installation/ilmt-track-bamlcns-metrictool.adoc (hunk 29 above)
[the remark this answers is not in the document]
Comment (veena-ux) [reply installation/ilmt-track-bamlcns-metrictool.adoc]: Content updated

File: installation/ilmt-track-bamlcns-metrictool.adoc (hunk 31 above)
[the remark this answers is not in the document]
Comment (veena-ux) [reply installation/ilmt-track-bamlcns-metrictool.adoc]: content updated

File: partials/attributes.adoc (hunk 32 above)
Comment (veena-ux) [reply partials/attributes.adoc:11]: Fixed

File: partials/attributes.adoc (hunk 33 above)
Comment (veena-ux) [reply partials/attributes.adoc:13]: Fixed as VERSION_80 for consistency purpose, no dot in between

File: installation/release-notes-main/release-notes-9-0-1.adoc (hunk 34 above)
Comment (veena-ux) [reply installation/release-notes-main/release-notes-9-0-1.adoc:20]: Fixed

File: installation/release-notes-main/release-notes-9-0-1.adoc (hunk 35 above)
Comment (veena-ux) [reply installation/release-notes-main/release-notes-9-0-1.adoc:71]: Fixed

File: installation/release-notes-main/release-notes-9-0-1.adoc (hunk 36 above)
Comment (veena-ux) [reply installation/release-notes-main/release-notes-9-0-1.adoc:74]: Fixed

File: installation/release-notes-main/release-notes-9-0-1.adoc [hunk 37]
```diff
@@ -17,7 +17,7 @@ The release notes contain information to ensure the successful installation and
 
 == Description
 
-{PRODUCT_LONG}, {VERSION901}  fix release, provides important bug fixes requested by customers and the security fixes to PSIRTs/CVEs.
+{PRODUCT_LONG}, {VERSION901}  fix release, provides important bug fixes requested by customers and the security fixes to CVEs.
```
Comment (LightGuard) [inline installation/release-notes-main/release-notes-9-0-1.adoc:20]: This should be VERSION_901, otherwise it will result in a warning during the build.

**Review by LightGuard — CHANGES_REQUESTED** [review]

**Review by LightGuard — APPROVED** [review]

**Review by porcelli — APPROVED** [review]
